projects/storefrontapp/src/environments/environment.ts

```diff
@@ -6,4 +6,5 @@
 export const environment = {
   production: false,
   occBaseUrl: '',
+  channel: 'b2b',
```

Comment:
I am not sure we want to default to b2b in our environment file. Is this something temporary?

Reply:
The default is 'b2c', but this feature branch is for b2b only. So, I set it 'b2b'. Before merge into develop, I will change it back to 'b2c'.